torch/_library/custom_ops.py

```diff
@@ -187,6 +191,54 @@ def _qualname(self) -> str:
     def __repr__(self) -> str:
         return f"<CustomOpDef({self._qualname})>"
 
+    @contextmanager
+    def disable_kernel(self, device_type: str, disable: bool = True):
```

Comment:
I think it's a little confusing for this to be named `disable_kernel` and also have a `disable` option. Either there should be 2 implementations (e.g. enable, disable), or use a different name (e.g. set_kernel_disabled?). I'm taking inspiration from `torch.no_grad, torch.enable_grad, torch.set_grad_enabled`.

Reply:
Maybe set_kernel_enabled(enabled=True/False) ?